crates/next-core/src/app_segment_config.rs

```diff
@@ -245,10 +245,8 @@ impl Issue for NextSegmentConfigParsingIssue {
     }
 
     #[turbo_tasks::function]
-    async fn source(&self) -> Result<Vc<OptionIssueSource>> {
-        Ok(Vc::cell(Some(
-            self.source.resolve_source_map().await?.into_owned(),
```

Comment:
Why was this change?

Reply:
I think I tried this in the past and @sokra said we should not do it.

Reply:
Looks fine to me. I can't remember the case, maybe it was moved into the other direction, which would be problematic...